astropy/coordinates/tests/test_frames.py

```diff
@@ -1532,3 +1532,16 @@ class Test:
     # This subclassing is the test!
     class NewFrame(ICRS, Test):
         pass
+
+
+def test_frame_coord_comparison():
+    """Test that frame can be compared to a SkyCoord"""
+    frame = ICRS("0d", "0d")
+    coord = SkyCoord(frame)
+    other = SkyCoord(ICRS("0d", "1d"))
+
+    assert frame == coord
+    assert frame != other
+
+    with pytest.raises(TypeError):
+        _ = frame == SkyCoord(AltAz("0d", "1d"))
```

Comment:
This is unnecessary.
```suggestion
        frame == SkyCoord(AltAz("0d", "1d"))
```

Reply:
It silences my linter from a "Unused expression warning"...